ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/table.utils.ts

```diff
@@ -305,3 +305,14 @@ export function formatVulnerabilityData(
         };
     });
 }
+
+// Given probability as float fraction, return as percent with 3 decimal digits.
+export function formatEpssProbabilityAsPercent(epssProbability: unknown) {
```

Comment:
`unknown` type here is due to hard coded `const epssProbability = undefined;` until the field is available in GraphQL queries, correct?

Reply:
Good question. I probably should have changed to `number | undefined` to support optional chaining both during implementation, but for some edge cases even after:
* Some CVE might not have a score
* RHSA without Scanner V4 might not have a score